src/github.com/stellar/go-horizon/resource_account.go

```diff
@@ -13,11 +16,17 @@ import (
 // AccountResource is the summary of an account
 type AccountResource struct {
```

Comment:
I think the attribute names in this struct should reflect the names used in the json encoding of this struct.  Given that:

`Numsubentries` should be `SubentryCount`
`InflationDest` should be `InflationDestination`